src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -89,7 +89,14 @@ protected function findTemplate($template, $throw = true)
         }
 
         if (false === $file || null === $file) {
-            throw new \Twig_Error_Loader(sprintf('Unable to find template "%s".', $logicalName), -1, null, $previous);
+            try {
+                list($namespace, $name) = $this->parseName($logicalName);
```

Comment:
I'm not sure what would be a good solution here. One could be to get the exception message from ```$previous```. \Twig_Loader_Filesystem throws exception if there are [no registered paths for namespace](https://github.com/twigphp/Twig/blob/master/lib/Twig/Loader/Filesystem.php#L197) or when [Unable to find template](https://github.com/twigphp/Twig/blob/master/lib/Twig/Loader/Filesystem.php#L216) with the given paths. These exception messages already filled with the ```$paths``` and ```$name```, what you'd like to extract here. TwigBundle's FilesystemLoader catch these exceptions and [saves it](https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php#L80) in a ```$previous``` variable. What could be done is just to check, if ```$previous``` is not null and get the exception message, ohterwise throw the exception with there original message.